sql/sql_plugin.cc

```diff
@@ -3237,8 +3240,12 @@ static int construct_options(MEM_ROOT *mem_root, st_plugin_int *tmp,
       (char *)mem_root->Alloc(plugin_name_len + plugin_dash.length + 1);
   strxmov(plugin_name_with_prefix_ptr, plugin_dash.str, plugin_name_ptr, NullS);
 
-  if (tmp->load_option != PLUGIN_FORCE &&
-      tmp->load_option != PLUGIN_FORCE_PLUS_PERMANENT) {
+  /* After changing static ROCKSDB plugin as mandatory, the --rocksdb option is
+   * not considered as loose. Add the option in the option list to avoid unknown
```

Comment:
Not a common MySQL comment style, the common ones are `//` and `/* */` without leading asterix in the intermediate lines